frontend/scripts/react-components/tool/tool.thunks.js

```diff
@@ -5,8 +5,4 @@ export const loadDisclaimerTool = dispatch => dispatch(loadDisclaimer());
 
 export const resizeSankeyTool = dispatch => dispatch(resize());
 
-export const loadToolInitialData = (dispatch, getState, bag) => {
```

Comment:
https://www.youtube.com/watch?v=UKfceBq-j9Y

If we remove this, then every time we update the url we're refetching stuff.